trade-finance-manager-ui/templates/utilisation-reports/_macros/selected-reported-fees-details-table.njk

```diff
@@ -0,0 +1,36 @@
+{% macro render(params) %}
+  {% set feeRecords = params.reportedFeeDetails.feeRecords %}
+  {% set totalReportedPayments = params.reportedFeeDetails.totalReportedPayments %}
+  <table class="govuk-table"
+         data-module="moj-sortable-table"
+         data-cy="selected-reported-fees-details-table">
+    <caption class="govuk-table__caption govuk-table__caption--m">Selected reported fees details</caption>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Facility ID</th>
+        <th scope="col" class="govuk-table__header">Exporter</th>
+        <th scope="col" class="govuk-table__header govuk-table__header--numeric">Reported fee</th>
+        <th scope="col" class="govuk-table__header govuk-table__header--numeric">Reported payments</th>
```

Comment:
Shouldn't this be `Reported payment`? Or should the above be `Reported fees`?

Reply:
That is what the screenshot of the design on my ticket has as the headers